x-pack/plugins/observability_solution/observability_onboarding/public/assets/auto_detect.sh

```diff
@@ -345,59 +354,82 @@ detect_known_integrations() {
   # System integration will still be able to to collect metrics.
   known_integrations_list_string+="system"$'\n'
 
-  local nginx_patterns=(
-    "/var/log/nginx/access.log*"
-    "/var/log/nginx/error.log*"
-  )
+  curl -L -O "${integration_config_file}" --silent --fail
+  config_file="./integrations.conf"
```

Comment:
You don't need to save the file to the local file system. 

You can assign the file contents directly to a variable:

```
integrations_config=$(curl "${kibana_api_endpoint}/integrations.conf" --fail)
```